src/index.js

```diff
@@ -44,7 +45,7 @@ export default function promiseMiddleware(config = {}) {
        * @returns {object} action
        */
       const getAction = (newPayload, isRejected) => ({
-        type: `${type}_${isRejected ? _REJECTED : _FULFILLED}`,
+        type: `${type}${promiseTypeSeparator}${isRejected ? _REJECTED : _FULFILLED}`,
```

Comment:
Perhaps `[type, isRejected ? _REJECTED' : _FULFILLED].join(promiseTypeSeparator)` would be better.